mqtt/mqtt-bridge/tests/integration.rs

```diff
@@ -221,7 +219,7 @@ async fn bridge_settings_update() {
 #[tokio::test]
 async fn subscribe_to_upstream_rejected_should_retry() {
     let (mut local_server_handle, _, mut upstream_server_handle, upstream_broker_handle) =
-        setup_brokers(AllowAll, DummySubscribeAuthorizer(false));
+        common::setup_brokers(AllowAll, DummySubscribeAuthorizer(false));
 
     let subs = vec![
```

Comment:
we duplicate these subs quite a bit, so we could define outside scope of an individual test for reuse

Reply:
while this is true for production code, for the test data it's better to have it inside the test instead.